Lib/idlelib/rpc.py

```diff
@@ -530,9 +530,8 @@ class RPCClient(SocketIO):
     nextseq = 1 # Requests coming from the client are odd numbered
 
     def __init__(self, address, family=socket.AF_INET, type=socket.SOCK_STREAM):
-        self.listening_sock = socket.socket(family, type)
-        self.listening_sock.bind(address)
-        self.listening_sock.listen(1)
+        self.listening_sock = socket.create_server(
+            address, family=family, type=type, backlog=1)
```

Comment:
Ouch! I missed that, thanks. I just removed this part and left idlelib alone in f786884. I will probably get back at this in a separate PR because it's more controversial.